### ISSUE TITLE
Update dependencies to remove "5 high severity vulnerabilities"

### ISSUE THREAD

**Issue by bluepuma77** [hitgeek/simple-hl7#83]
We get "5 high severity vulnerabilities" when running `npm install simple-hl7` and  `npm audit report`:

```
# npm audit report

merge  <2.1.1
Severity: high
Prototype Pollution in merge - https://github.com/advisories/GHSA-7wpw-2hjm-89gp
fix available via `npm audit fix --force`
Will install simple-hl7@2.2.1, which is a breaking change
node_modules/merge
  exec-sh  <=0.3.1
  Depends on vulnerable versions of merge
  node_modules/exec-sh
    watch  >=0.14.0
    Depends on vulnerable versions of exec-sh
    node_modules/watch
      simple-hl7  1.0.0 - 3.2.4
      Depends on vulnerable versions of moment
      Depends on vulnerable versions of watch
      node_modules/simple-hl7

moment  <=2.29.3
Severity: high
Path Traversal: 'dir/../../filename' in moment.locale - https://github.com/advisories/GHSA-8hfj-j24r-96c4
Inefficient Regular Expression Complexity in moment - https://github.com/advisories/GHSA-wc69-rhjr-hc9g
fix available via `npm audit fix --force`
Will install simple-hl7@2.2.1, which is a breaking change
node_modules/moment

5 high severity vulnerabilities
```

**Comment by bluepuma77** [comment]
```
# npm install simple-hl7 --force
# npm audit

merge  <2.1.1
Severity: high
Prototype Pollution in merge - https://github.com/advisories/GHSA-7wpw-2hjm-89gp
fix available via `npm audit fix --force`
Will install simple-hl7@2.2.1, which is a breaking change
node_modules/merge
  exec-sh  <=0.3.1
  Depends on vulnerable versions of merge
  node_modules/exec-sh
    watch  >=0.14.0
    Depends on vulnerable versions of exec-sh
    node_modules/watch
      simple-hl7  1.0.0 - 3.2.4
      Depends on vulnerable versions of moment
      Depends on vulnerable versions of watch
      node_modules/simple-hl7

moment  <=2.29.3
Severity: high
Path Traversal: 'dir/../../filename' in moment.locale - https://github.com/advisories/GHSA-8hfj-j24r-96c4
Moment.js vulnerable to Inefficient Regular Expression Complexity - https://github.com/advisories/GHSA-wc69-rhjr-hc9g
fix available via `npm audit fix --force`
Will install simple-hl7@2.2.1, which is a breaking change
node_modules/simple-hl7/node_modules/moment
```

@hitgeek Could you just run `ncu -u` ([npm-check-updates](https://www.npmjs.com/package/npm-check-updates)) to update all dependencies to the latest version?

**Comment by hitgeek** [comment]
last time I reviewed this there was no way to remove all 5 vulnerabilities due to the `watch` package seemingly being unmaintained. i also didn't see any means to exploit these vulnerabilities via this library. so I chose to leave v3 as is. so while the vulnerabilities may be "high" in general, I don't considered them to be high in the context of this library. 

v4 is available via `npm install simple-hl7@next` has no current vulnerabilities. however due to the re-write of the file server to remove the watch dependency i have this as a new major version. i also added some new features that are currently in a beta state. 
i don't have any plans to promote v4 to the default version in the near future, due to limited bandwith to ensure full compatibility or clearly document breaking changes. 
i'm leaving it up to consumers to choose which to use based on their needs.
